pkg/cloudencrypt/log.go

```diff
@@ -0,0 +1,47 @@
+package cloudencrypt
+
+import (
+	"context"
+	"log"
+)
+
+// LogEncryptor wraps another Encryptor and adds logging.
+type LogEncryptor struct {
+	encryptor Encryptor
+	logger    *log.Logger
```

Comment:
This logger is very bad, try to consider to use some better logger than standard library one. There are plenty of them.
Ideally some leveled one e.g Debug/Info/Warn/Error

Reply:
In practice I'll make a different implementation in keboola-as-code with the logger we have there (if we actually need one which I'm not sure we do). I just needed this simple one for tests in order to check when the cached encryptor is actually calling the inner one.